libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorRecognizer.cs

```diff
@@ -27,13 +27,13 @@ public OrchestratorRecognizer()
         /// Gets or sets the id for the recognizer.
         /// </summary>
         /// <value>
-        /// The id for the recognizer.
+        /// The id for the recognizer.  Useful for looking up specific recognizer in multiple recognizers scenario.
```

Comment:
Is this just an arbitrary identifier or does it need to match something?  Where/how do you look it up?

Reply:
Just arbitrary.  It's actually in the Recognizer class, which OrchestratorAdaptiveRecognizer is based on, but not OrchestratorRecognizer (quite strange!).  I wanted to ask John, who reviewed Vishwac's PR but he's OOF.  Regardless, we don't want to change the code right now.